### ISSUE TITLE
Feature request: Adjustable timescale for graphs

### ISSUE THREAD

**Issue by ghost** [makerdao/governance-dashboard#1]
Hey Hernando, the dashboard looks awesome.

Would it be possible to allow users to set custom date ranges for the graphs?

Also a button on the "At selected time" graph to go back to now/today would be nice :)

**Comment by hernandoagf** [comment]
Yeah, totally possible :) Did you have in mind a general time controller for all the dashboard or individual controllers for each chart?

**Comment by ghost** [comment]
I was imagining it for each chart but either sounds good!
